### ISSUE TITLE
Security style

### ISSUE THREAD

**Issue by olivibernard** [NetApp/netappdvp#50]
Hi,

When I create a volume, I can not specify the security_style option.
I tried something like this : 
```
docker volume create -d netapp  --name vol1 -o security-style=unix
docker volume create -d netapp  --name vol2 -o securityStyle=unix
```
None of them works.

I read the **azgo/volume-create.go**  file. This source seems to use  but I can not figue out what to use in the command line.

The problem is the volume is created, but the security_style is NTFS instead of UNIX.
I get the default value which is configured by the netapp admin guy.
The result is I can mount the created volume in a container, but I can not read/write it.

Is there a way to force this option (UNIX) in `docker volume create` options ?

**Comment by ntap-rippy** [comment]
Need to add a **securityStyle** parameter and function call here
[apis/ontap/ontap.go#VolumeCreate](https://github.com/NetApp/netappdvp/blob/66a14202f8337cc19f4e8c9b928bfd496860e40a/apis/ontap/ontap.go#L165-L176)

Need to add the **securityStyle**  option here
[storage_drivers/ontap_nas.go#Create](https://github.com/NetApp/netappdvp/blob/66a14202f8337cc19f4e8c9b928bfd496860e40a/storage_drivers/ontap_nas.go#L146-L205)

**Comment by adkerr** [comment]
Internally tracked by [US2201](https://rally1.rallydev.com/#/47479883219d/detail/userstory/90428080320)

**Comment by innergy** [comment]
Fixed in 17.04.0
